spec/features/tslr_claim_spec.rb

```diff
@@ -22,7 +22,52 @@
     click_on "Continue"
 
     expect(claim.reload.claim_school).to eql schools(:penistone_grammar_school)
-    expect(page).to have_text("Are you still employed to teach at a school in the UK")
+    expect(page).to have_text("Are you still employed to teach at a school in England")
+
+    choose "Yes, at Penistone Grammar School"
+    click_on "Continue"
+
+    expect(claim.reload.employment_status).to eql("claim_school")
+    expect(claim.current_school).to eql(schools(:penistone_grammar_school))
+    expect(page).to have_text("Did you teach")
+  end
+
+  scenario "Teacher now works for a different school" do
+    visit root_path
+
+    click_on "Agree and continue"
+
+    claim = TslrClaim.order(:created_at).last
+
+    expect(page).to have_text("Which academic year were you awarded qualified teacher status")
+    select "September 1 2014 - August 31 2015", from: :tslr_claim_qts_award_year
+    click_on "Continue"
+
+    expect(claim.reload.qts_award_year).to eql("2014-2015")
+    expect(page).to have_text("Which school were you employed at between")
```

Comment:
As this spec is testing the different school question, do you think making the assertions above make our tests more brittle? We already cover them in our main 'happy path' spec above this one.
If for some reason we weren't on the right page at this point we would fail anyway as the form fields would be missing.

Reply:
Rather than testing the question like a unit, I feel like this test is to give us confindence that the journey for a user with a different school is working end to end.

We talked about removing the duplication in these tests but decided to let it stand for now as the feature test suite grows we feel like we'll reach a point that this will get refactored.

Reply:
Yeah I agree with testing it end-to-end, I was just curious about duplicating the `expect` parts. Happy to leave it for now and look at refactoring later if it needs it